README.md

```diff
@@ -12,51 +12,139 @@ decrypted only by the controller running in the target cluster and
 nobody else (not even the original author) is able to obtain the
 original Secret from the SealedSecret.
```

Comment:
I'd add a wrap-up line on the lines of: 

End result is that you get a stock Secret (usable from a e.g. Pod manifest) after pushing the SealedSecret having the same namespace and name.

Reply:
I think there's some overlap between this and my comment here, https://github.com/bitnami-labs/sealed-secrets/pull/231/files#r319470607.

Reply:
@jjo thanks; makes sense. Is this similar to what matt said in https://github.com/bitnami-labs/sealed-secrets/pull/231#discussion_r319470607 ?

is it ok how addresses it there?

Reply:
Sure, thanks much for this update.